topics/dev/tutorials/interactive-tools/tutorial.md

```diff
@@ -472,6 +489,16 @@ our new Docker container as a Galaxy tool.
 >    >         Some help is always of interest ;)
 >    >
 >    >     ]]></help>
+>    >     <citations>
+>    >        <citation type="bibtex">
+>    >        @misc{
+>    >             author       = {Lain Pavot - lain.pavot@inrae.fr },
+>    >             title        = {{first-gxit -  A tool to visualise tsv/csv files }},
```

Comment:
These double braces need to be escaped with a `{% raw %}` tag - see next comment

Reply:
Thanks for the tip